WebApp/autoreduce_webapp/reduction_viewer/models.py

```diff
@@ -46,6 +46,17 @@ def __unicode__(self):
         return u'%s' % self.value
 
 
+class Software(models.Model):
+    """
+    Represents the software used to perform the reduction
+    """
+    name = models.CharField(max_length=100, blank=False)
+    version = models.CharField(max_length=20, blank=False)
```

Comment:
Based on my comment in `model/database/tests/test_access.py` about not None validation, we could alternatively just add not null validation to both columns in the model itself unless there is a need to have just the name or just the version column populated for a given row.